src/test/java/com/anusha/testing/AppTest.java

```diff
@@ -42,4 +51,318 @@ public void shouldMultiplyTwoIntegers(){
+    System.out.println("Unsorted ArrayList: " + unsortedArrayList);
+    ArrayList<Integer> sortedArrayListAscending = sortArrayList.sortAscending();
+    System.out.println("Sorted ArrayList in Ascending Order : " + sortedArrayListAscending);
+    ArrayList<Integer> sortedArrayListDescending = sortArrayList.sortDescending();
+    System.out.println("Sorted ArrayList in Descending Order: " + sortedArrayListDescending);
+}
+
+// convert from wrapper to primitive and display ascending and descending order
+@Test
+public void shouldDisplayAscendingOrderArraylistPrimitive(){
+
+
+         ArrayList<Integer> myArr = new ArrayList<>();
+    myArr.add(15);
+    myArr.add(98);
+    myArr.add(75);
+    myArr.add(59);
+    myArr.add(88);
+    myArr.add(9);
+    myArr.add(25);
+
+    //alt. mtd
+
+       /*Integer[] array=new Integer[]{15,98,75,59,88,9,25};
+        List<Integer> myArr=Arrays.asList(array); */
+
+    List<Integer> ascendingOrder=ArrayUtils.getAscendingOrderElementList(myArr);
+
+    // Integer actualLowest = myArr.get( myArr.size() - ) ; ;
+    int actualHighest = myArr.get( myArr.size() - 1) ;
+    int actualLowest = myArr.get( 0 ) ;;
+    int expectedLowest=9;
+    int expectedHighest=98;
+
+    Assert.assertEquals(expectedLowest,actualLowest);
+    Assert.assertEquals(expectedHighest,actualHighest);
+
+}
+/*
+ @Test
+    public void  shouldCheckElementsCopied(){
+     ArrayList<Integer> arrayList1 = new ArrayList<>();
+
+     arrayList1.add(100);
+     arrayList1.add(200);
+     arrayList1.add(300);
+
+     ArrayList arrayList2 = new ArrayList();
+
+     arrayList2.add(400);
+     arrayList2.add(500);
+     arrayList2.add(600);
+     arrayList2.add(700);
+     arrayList2.add(800);
+
+List<Integer> actual=ArrayUtils.getCopiedElementsIntoAnotherList();
+List<Integer> expected=
+        assertThat(actual.getList(), containsInAnyOrder(expectedList.toArray(new String[expectedList.size()]));
+
+ }  */
+
+ @Test
+    public void shouldCheckForStringListLength(){
+     List<String> myString=new ArrayList<>();
+     myString.add("Anusha");
+     myString.add("Gopal");
+     myString.add("Goutham");
+     myString.add("Sriraksha");
+
+     int actual=myString.size();
+     int expected=4;
+
+     Assert.assertEquals(expected,actual);
+ }
+
+ @Test
+    public void shouldCheckForAddedString(){
+     String a="Goutham";
+     String b="Rao";
+     String c=a+b;
+
+     String actual=c;
+     String expected="GouthamRao";
+
+     //Assert.assertEquals(expected,actual);
+     assertThat(c, is("GouthamRao"));
+ }
+
+ @Test
+    public void shouldDisplayUppercasedString(){
+
+     String str="anusha";
+     String actual=str.toUpperCase();
+     String expected="ANUSHA";
+     assertEquals(expected,actual);
+
+ }
+
+ @Test
+ public void shouldDisplayParsedIntegerValue(){
+     int number=2;
+     Integer actual=Integer.valueOf(number);
+     Integer expected=2;
+     assertEquals(expected,actual);
```

Comment:
parsing means reading from String... so you be converting a string to an integer... String s = "4";
Integer i = convert s to Integer... look for Integer.parse method